distribution/std-lib/Base/src/Vector.enso

```diff
@@ -161,6 +161,29 @@ type Vector
             arr.set_at i (that.at i-this_len)
         Vector arr
 
+    ## When `this` is a vector of text values, concatenates all the values by
+       interspersing them with `separator`.
+
+       > Example
+         The following code returns "foo, bar, baz"
+             ["foo", "bar", "baz"].join ", "
+    join : String -> Text
+    join separator =
+        if this.length == 0 then "" else
+            if this.length == 1 then this.at 0 else
+                this.at 0 + (1.upto this.length . fold "" acc-> i-> acc + separator + this.at i)
+
+    ## Returns a new vector, consisting of all but the first `count` elements
+       of `this`.
+    drop : Integer -> Vector
+    drop count = if count >= this.length then Vector.new 0 (x -> x) else
+        Vector.new (this.length - count) (i -> this.at i+count)
```

Comment:
Given that `drop` is likely an unfamiliar notion to novices, I'd suggest we call this `drop_left` and have a dual `drop_right`.

Reply:
Many of these are unfamiliar, this is why we have docs. I don't want to make this API super-verbose for no apparent reson.

Reply:
Well then you could make the docs clearer and mentinoing that it drops from the left. I'd also like to have the duals added.

Reply:
What's unclear? "All but the first `count` elements" seems like reasonable explanation to me.